apps/pm-user/src/components/molecules/resultSection/ResultSection.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, Result } from '../../atoms';
+import { useHistory } from 'react-router-dom';
+import { HomeOutlined } from '../../atoms/icon';
```

Comment:
Yes. I agree with you.
But Icons are special and are not normal components. 
So for the sake of easy identification, I am deliberately importing it from "atoms/icons" instead of "atoms" directly.
Your logic is correct. But only for the sake of ease of understanding what is what... I've imported from "atoms/icons" folder instead of "atoms/" directly

Reply:
I remember Chaitanya doing it in a similar way. Hence I did it.

Reply:
No chetanaya's way is this
import { HomeFilled } from '../../../atoms';

you can check it in
oraganism -> sideBar -> constanece -> menuItems.js

Reply:
Okay. Changing this.